data/_models/shared/Contacts.csv

```diff
@@ -1,19 +1,19 @@
-Contacts,,homepage,,,,,,,,,foaf:workplaceHomepage,Link to contact's homepage,"DataCatalogueFlat,CohortsStaging,RWEStaging,NetworksStaging,StudiesStaging,UMCGCohortsStaging",
-Contacts,,photo,file,,,,,,,,foaf:img,Contact's photograph,"DataCatalogueFlat,CohortsStaging,RWEStaging,NetworksStaging,StudiesStaging,UMCGCohortsStaging",
-Contacts,,expertise,,,,,,,,,,Description of contact's expertise,"DataCatalogueFlat,CohortsStaging,RWEStaging,NetworksStaging,StudiesStaging,UMCGCohortsStaging",
\ No newline at end of file
+Contacts,,,,,,,,,,,vcard:Kind,Listing of contact persons per resource,"DataCatalogueFlat,EMA,CohortsStaging,RWEStaging,NetworksStaging,StudiesStaging,UMCGCohortsStaging",,
+Contacts,,resource,ref,1,TRUE,,Resources,,,,,Resource the contact is affiliated with,"DataCatalogueFlat,EMA,CohortsStaging,RWEStaging,NetworksStaging,StudiesStaging,UMCGCohortsStaging",,
+Contacts,,role,ontology_array,,,CatalogueOntologies,Contribution types,,,,vcard:hasRole,Type(s) of contribution or role in the resource,"DataCatalogueFlat,EMA,CohortsStaging,RWEStaging,NetworksStaging,StudiesStaging,UMCGCohortsStaging",,
+Contacts,,role description,text,,,,,,,,,Description of the role,"DataCatalogueFlat,CohortsStaging,RWEStaging,NetworksStaging,StudiesStaging",,
+Contacts,,first name,,1,TRUE,,,,,,foaf:firstName,First name of the contact person,"DataCatalogueFlat,EMA,CohortsStaging,RWEStaging,NetworksStaging,StudiesStaging,UMCGCohortsStaging",,
+Contacts,,last name,,1,TRUE,,,,,,foaf:familyName,Last name of the contact person,"DataCatalogueFlat,EMA,CohortsStaging,RWEStaging,NetworksStaging,StudiesStaging,UMCGCohortsStaging",,
+Contacts,,display name,,,TRUE,,,,,,"foaf:name,vcard:hasName",Display name of the contact person,DataCatalogueFlat,FALSE,firstName+' '+lastName
+Contacts,,statement of consent personal data,bool,,TRUE,,,,,,,"Indicate whether this person consents to display the following personal data in the catalogue (which is openly accessible): title, name, photo, orcid ID, institute, department, expertise, discipline and role within the resource. These data-items may partly be extracted from the UMCG and/or RUG employee pages. The consent also includes future linkages with other national and international catalogues and platforms to increase visibility and findability (e.g. BBMRI-ERIC, NARCIS) of UMCG-resources. Contributors without a positive statement of consent will not be shown in the UMCG Research Data Catalogue",UMCGCohortsStaging,,
+Contacts,,prefix,,,,,,,,,,"Surname prefix, if applicable","DataCatalogueFlat,CohortsStaging,RWEStaging,NetworksStaging,StudiesStaging,UMCGCohortsStaging",,
+Contacts,,initials,,,,,,,,,,Initials of the contact person,"DataCatalogueFlat,CohortsStaging,RWEStaging,NetworksStaging,StudiesStaging,UMCGCohortsStaging",,
+Contacts,,title,ontology,,,CatalogueOntologies,Titles,,,,foaf:title,Title of the contact person,"DataCatalogueFlat,EMA,CohortsStaging,RWEStaging,NetworksStaging,StudiesStaging,UMCGCohortsStaging",,
```

Comment:
you want to do the 'pre' and 'post' titles later?

Reply:
They all go under 'title' I think? 'prefix' is meant for e.g. 'van der' in 'van der Velde'

Reply:
ah no. You have titles before, such as 'Prof.dr.' and there are pre-nominal letters such as 'PhD'
see https://rapidbi.com/post-nominal-letters-and-qualifications/
but I don't mind to revisit this later.

Reply:
Yes I understood. I would like to do this later, it is not within the scope of the migrations.

Reply:
I made a separate issue for this here: https://github.com/molgenis/GCC/issues/716

Reply:
great :)